### PR TITLE
[ICDS Dashboard] Change title of usage report csv

### PR DESCRIPTION
**Pull request by challabeehyv**
https://dimagi-dev.atlassian.net/browse/QA-859

##### SUMMARY
Change title of usage report csv

##### FEATURE FLAG
No

### CODE REVIEW

File: custom/icds_reports/sqldata/exports/dashboard_usage.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 class DashBoardUsage:
 
-    title = 'Dashboard Activity'
```
Comment (Rohit25negi) [inline custom/icds_reports/sqldata/exports/dashboard_usage.py]: @challabeehyv  I think this is also used as the tab name in the excel sheet can you check with QA/Saket whether that also needs to changed to this or not. ?

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Hey, @challabeehyv  looks good. left one question

File: custom/icds_reports/sqldata/exports/dashboard_usage.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: QA wanted consistency in the name everywhere. So changed it accordingly. Let me confirm with QA team

Comment (challabeehyv) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: @Rohit25negi Confirmed this as a requirement from QA.

Comment (Rohit25negi) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: you mean title of the tab in excel report also needs to be changed along with csv file name?

Comment (challabeehyv) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: @Rohit25negi latest : file names should have 'Report' suffix.

Comment (challabeehyv) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: > @challabeehyv I think this is also used as the tab name in the excel sheet can you check with QA/Saket whether that also needs to changed to this or not. ?

we are setting the tab name separately

Comment (challabeehyv) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: > you mean title of the tab in excel report also needs to be changed along with csv file name?

in short, excel tab should have Dashboard Activity and csv file should have Dashboard Activity Report

Comment (Rohit25negi) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: I see

**Review by Rohit25negi — APPROVED** [review]
Looks good. good to merge once the test cases passes

**Pull request merged**